.gitmodules

```diff
@@ -14,3 +14,6 @@
 	path = translations/docs-zh
 	url = https://github.com/dapr/docs-zh.git
 	branch = v1.0_content
+[submodule "sdkdocs/go"]
+	path = sdkdocs/go
+	url = git@github.com:dapr/go-sdk.git
```

Comment:
The differing format of git@ instead of https:// was causing some issues when I attempted to initialize the submodules 
![image](https://user-images.githubusercontent.com/54363786/124186270-ef3ee180-da70-11eb-8cd4-7ec76424b5e7.png)

Reply:
Oh that's interesting. I'll switch it to https and see what happens